pythonforandroid/recipe.py

```diff
@@ -1056,12 +1160,14 @@ def prebuild_arch(self, arch):
             exit(1)
         self.ctx.python_recipe = self
 
-    def include_root(self, arch):
+    def include_root(self, arch_name):
         '''The root directory from which to include headers.'''
-        raise NotImplementedError('Not implemented in TargetPythonRecipe')
+        return join(self.get_build_dir(arch_name),
+                    'Include')
```

Comment:
Minor nitpick: here and other place, it looks like some very agressive line length, isn't it?